source/operations/install-deploy-manage/deploy-operator-kustomize.rst

```diff
@@ -0,0 +1,226 @@
+.. _minio-k8s-deploy-operator-kustomize:
+
+==============================
+Deploy Operator With Kustomize
+==============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+`Kustomize <https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization>`__ is a YAML-based templating tool that allows you to define Operator and Tenant templates using plain Kubernetes YAML reference language.
+Kustomize is included with the `kubectl <https://kubernetes.io/docs/reference/kubectl/>`__ command line tool.
+
+The `default MinIO Operator Kustomize template <https://github.com/minio/operator/blob/master/kustomization.yaml>`__ provides a starting point for customizing configurations for your local environment.
+You can modify a Kustomize Operator deployment after installation with kubectl patches, which are additional YAML configurations applied over the existing base.
+
+
+Prerequisites
+-------------
+
+To install the Operator with Kustomize you will need the following:
+
+* An existing Kubernetes cluster, v1.21 or later.
+* The ``kubectl`` CLI tool on your local host, the same version as the cluster.
+* `Docker <https://docker.com>`__ for your platform.
```

Comment:
:thinking: does the local host need docker?

Reply:
🤔 a good point. My setup uses it. But that doesn't really mean much, does it.